### PR TITLE
Fix rspec pinning

### PR DESCRIPTION
**Pull request by shortdudey123**
Remove rspec version pinning and update stub syntax

### CODE REVIEW

**Comment by johnsyweb** [comment]
LGTM

**Comment by shortdudey123** [comment]
Fixed merge conflicts

File: spec/pagerduty_spec.rb [hunk 1]
```diff
@@ -229,17 +238,19 @@
...
           Given {
-            transport.stub(:send_payload => {
-              "status" => "failure",
-              "message" => "Event not resolved",
-            })
+            allow(transport).to receive(:send_payload).and_return(
+              {
+                "status" => "failure",
+              "message" => "Event not resolved"
+              }
```
Comment (orien) [inline spec/pagerduty_spec.rb]: These `{ }` aren't required here. IMO the code reads better without them. (minor indentation issue here also.)
```ruby
 allow(transport).to receive(:send_payload).and_return(
  "status" => "failure",
  "message" => "Event not resolved",
)
```
Similar comment for above changes.

**Comment by orien** [comment]
Thanks!

**Pull request merged**